pyControl4/director.py

```diff
@@ -209,3 +216,7 @@ async def getItemBindings(self, item_id):
             `item_id` - The Control4 item ID.
         """
         return await self.sendGetRequest("/api/v1/items/{}/bindings".format(item_id))
+
+    async def getUiConfiguration(self):
+        """Get the JSON Control4 App UI Configuration enumerating rooms and capabilities"""
```

Comment:
Can you provide an example of what kind of output this returns?

See here for an example: https://github.com/lawtancool/pyControl4/blob/065485fea8bcac6e4db0d3024596e3fdc1550630/pyControl4/account.py#L157

Reply:
Done.